coursefunctions.h

```diff
@@ -172,30 +172,42 @@ void TestDrawTriangle(Buffer2D<PIXEL> & target)
         verts[2] = {50, 452, 1, 1};
         PIXEL colors1[3] = {0xffff0000, 0xffff0000, 0xffff0000};
         // Your color code goes here for 'attr'
+        attr[0].color = colors1[0];
```

Comment:
minor: you could have just assigned the array to one color because every color is the same in the array